scripts/create_network.py

```diff
@@ -82,12 +116,23 @@ def create_microgrid_network(n, input_file, number_microgrids):
             bus0 = microgrid_buses.index[i]
             bus1 = microgrid_buses.index[j]
             line_name = f"{microgrid_id}_line_{i}_{j}"
-            n.add("Line", line_name, bus0=bus0, bus1=bus1, x=0.01, r=0.1)
+            n.add(
+                "Line",
+                line_name,
+                bus0=bus0,
+                bus1=bus1,
+                x_per_length=0.335,
+                r_per_length=1.2012,
+                c_per_length=11.25,
+                i_nom=0.14,
+                type="24-AL1/4-ST1A 0.4",
```

Comment:
Type shall be by config, likewise voltage level.

With the linetipe, you don't need the x/r/c. Moreover, with madd function this iterative adding is not needed